Marlin/Marlin_main.cpp

```diff
@@ -104,6 +104,7 @@
  * M1   - Same as M0
  * M17  - Enable/Power all stepper motors
  * M18  - Disable all stepper motors; same as M84
+ * M19  - Resume print from current (or given) Z height (disables all movements below the current Z position, a file must be selected to print after executing this M code)
```

Comment:
I wonder if we can do this by adding a `Z` parameter to `M24` instead, since it is the command for both Print and Resume. But I could see M19 being useful in cases where the host is printing a file - but some hosts have their own Resume functions, I believe.